consensus/types/src/config_and_preset.rs

```diff
@@ -50,7 +53,26 @@ impl ConfigAndPreset {
         let capella_preset = CapellaPreset::from_chain_spec::<E>(spec);
         let extra_fields = get_extra_fields(spec);
 
-        if spec.electra_fork_epoch.is_some()
+        if spec.fulu_fork_epoch.is_some()
+            || fork_name.is_none()
+            || fork_name == Some(ForkName::Fulu)
+        {
+            let deneb_preset = DenebPreset::from_chain_spec::<E>(spec);
```

Comment:
I think we can move `deneb_preset` to the top and drop support for returning `ConfigAndPresetCapella`?